bin/spring

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+# This file loads spring without using Bundler, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
+
+unless defined?(Spring)
+  require "rubygems"
+  require "bundler"
+
+  if match = Bundler.default_lockfile.read.match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m)
```

Comment:
__[development][rubocop]__
* Lint/AssignmentInCondition: Assignment in condition - you probably meant to use `==`. [:ref](https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition)
* Metrics/LineLength: Line is too long. [95/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)